scaleway/data_source_domain_zone_test.go

```diff
@@ -4,14 +4,19 @@ import (
 	"fmt"
 	"testing"
 
+	"github.com/hashicorp/go-uuid"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 )
 
 func TestAccScalewayDataSourceDomainZone_Basic(t *testing.T) {
 	tt := NewTestTools(t)
 	defer tt.Cleanup()
 
-	testDNSZone := "test-zone"
+	id, err := uuid.GenerateUUID()
```

Comment:
oops... if you put a random uuid every time the acceptances test will never pass 😢